.travis.yml

```diff
@@ -25,11 +25,22 @@ jobs:
       after_success:
         - xcrun llvm-cov export -format="lcov" .build/debug/ParseSwiftPackageTests.xctest/Contents/MacOS/ParseSwiftPackageTests -instr-profile .build/debug/codecov/default.profdata > info.lcov
         - bash <(curl https://codecov.io/bash) 
-        - gem install jazzy
-        - ./Scripts/jazzy.sh
     - stage: Package Managers
       env: Cocoapods
       script: pod lib lint
     - stage: Package Managers
       env: Carthage
       script: carthage build --no-skip-current
+    - stage: Package Managers
+      script:
+        - gem install jazzy
+        - ./Scripts/jazzy.sh
+      deploy:
+        - provider: pages
+          skip_cleanup: true
+          token:
+            secure: ""
```

Comment:
@TomWFox can you add/setup the token?

Reply:
You haven't got maintainer edits turned on, here is the secure var - 

bW1tnO9sz1ehtyK4IbcjEUzDg7WvpMxd6wBz1BDxNi0IA4Qlv7KEd3+7gDL6xeNLue6OyVmCrLRoGpLQZmab21/FsX3lx87IV4xUJrg6wmruyUBtPWrTaHMTRHBsJFONsY0ADAn/MxXyVJyoK38gfnAUT6Q5WdfkTcn3jbB0sAqPjbd8JNJpfw4cAsxJjFL4Y4NDYIS8FWw/3uaEyX6JlNi6WOuomt3JKld91ioGH1gyAPoSDMufJPAm63jCGp/ichs4e/xLZ2XOziqgHZ6ujie9cTEf7cPk+xzzr1kuLFRMF9TPjO3Tf9MARK/vDoAMlR4a1h2q7wc6ok6FXGT37/01L7sX6JPzMBM8M5GJJpCaDIHj0PNG5QaN291kS7Vi7flLC8+ZkCddcPKh89m2uh7/FLNi3xh9RTazG0nq/Fztm3LlKzBRenF7aO9DCoOC2PFhg5vGuQk8N7SvK1A2kygjSJcj8gRqB3D0XnrcoNkXDiNjjgC01vhKOoHroTT0/Ds+KP0k6n9C/Hra0L5my7w0qnIkMffp6DW/VxMp3k+ZEJ4ZhVwCZ3pXJyon92tiqBweR0c0BjRpwyggJqIWxOeDJAM7v1Q6MoI1dO1sFQz7fDLh1VhcJpG4d7sg6J6mykotMDEd5re6VWwlpWf3ts7kYwldblq6SWr2Jk4yfL0=

Reply:
@TomWFox I'm thinking GitHub might have temporarily disabled "all maintainer edits" as I don't see it on the PR like how I use to on others and I usually leave that box checked

Reply:
Oh, maybe - anyway you can branch from the main repo now you've got write access